apps/backend/src/statusActionEngine/emailActionHandler.ts

```diff
@@ -88,6 +89,55 @@ export const emailActionHandler = async (
           break;
         }
 
+        case EmailStatusActionRecipients.USER_OFFICE: {
+          const userOfficeEmail = process.env.USER_OFFICE_EMAIL;
+
+          if (userOfficeEmail == null) {
```

Comment:
I think it is better to use `===` triple equal when doing this comparison and another question is what happens if `userOfficeEmail` is `undefined`.  Is there any specific reason why we compare it only against `null` and not using something like: `if (!userOfficeEmail) {`

Reply:
It would have checked for null and undefined with `==` due to type coercion. `===` seems a bit more verbose as we have to explicitly check both like `(userOfficeEmail === null || userOfficeEmail === undefined)`. But using `(!userOfficeEmail)` as you said sounds best as it will then additionally check for falsy values like an empty string.